src/debasmita/A20_StringCode1.java

```diff
@@ -0,0 +1,22 @@
+package debasmita;
+
+public class A20_StringCode1 {
+	
+	 int sum=0;
```

Comment:
if this variable is common for other methods then we should write it at the instance level.